src/java.base/share/classes/java/lang/foreign/Linker.java

```diff
@@ -244,6 +244,8 @@
  * </li>
  * </ul>
  *
+ * Linker implementations may optionally support additional layouts, such as 'packed' struct layouts.
```

Comment:
Optional: should we say what a packed struct layout is? E.g. make `packed` italic. Then follow up with a sentence like: 
```
a packed struct is a struct in which there is at least one member layouts L whose offsets does not respect L's natural alignment: <example>
```

Reply:
Yes, good idea

Reply:
See https://github.com/openjdk/jdk/pull/16485/commits/fea8a26325b3e6142c652821e0ef95d78992b36c

I've avoided talking about offset, since packing can also be used to avoid trailing padding.